.github/workflows/tests.yml

```diff
@@ -14,18 +14,20 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        resolver: [lts-16, lts-15, lts-14]
+        resolver: [lts-16, lts-15, lts-14, lts-12]
+        yaml: ['stack.yaml', 'stack-master.yaml']
 
     steps:
       - name: Clone project
         uses: actions/checkout@v2
 
       - name: Cache dependencies
-        uses: actions/cache@v1
+        uses: actions/cache@v2
         with:
           path: ~/.stack
-          key: ${{ runner.os }}-${{ matrix.resolver }}-${{ hashFiles('stack.yaml') }}
+          key: ${{ runner.os }}-${{ matrix.resolver }}-${{ hashFiles(matrix.yaml) }}
           restore-keys: |
+            ${{ runner.os }}-${{ matrix.resolver }}-${{ hashFiles(matrix.yaml) }}
```

Comment:
This should be a no-op. Is it not? If no, it looks like a hack and probably deserves a comment.

Reply:
@liskin Ah, I see what you mean by no-op here. I have pushed a commit removing this.